modules/ROOT/pages/flow-counting-studio.adoc

```diff
@@ -0,0 +1,29 @@
+= Tracking the Number of Billable Flows in Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In Anypoint Studio 7.20 and later, you can track the number of billable flows your project has.
+
+include::anypoint-code-builder::partial$acb-studio-shared-flow-counting.adoc[tag="flow-counting-intro"]
+
+include::anypoint-code-builder::partial$acb-studio-shared-flow-counting.adoc[tag="flow-counting-billable-flows"]
+
+== Before You Begin
+
+include::anypoint-code-builder::partial$acb-studio-shared-flow-counting.adoc[tag="flow-counting-prereq"]
+
+== Counting the Flows of Your Project in Studio
+
+. Select your project.
+. Click the *Flow Count* icon (image:flow-count-icon.png[2,2%""]) in the task bar.
```

Comment:
```suggestion
. Click *Flow Count* (image:flow-count-icon.png[2,2%""]) in the taskbar.
```

Reply:
I think it's taskbar, but our docs have more instances of "task bar" so <shrug>

Reply:
It's "taskbar". Great catch! Thanks!